sale_blanket_order/models/blanket_orders.py

```diff
@@ -264,9 +264,24 @@ def onchange_partner_id(self):
             values["team_id"] = self.partner_id.team_id.id
         self.update(values)
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        translated_draft = _("Draft")
+        for vals in vals_list:
+            if (
+                self.env.company.enable_numbered_bo
+                and vals.get("name", translated_draft) == translated_draft
+            ):
+                vals["name"] = (
+                    self.env["ir.sequence"].next_by_code("sale.blanket.order")
+                    or translated_draft
+                )
+        result = super().create(vals_list)
```

Comment:
**issue**: `action_confirm` wasn't changed.

The behaviour right now, with the setting activated, is that:
- I create a BO --> It takes number BO0001
- I confirm it --> It takes number BO0002